data/json/items/tool/lighting.json

```diff
@@ -1081,5 +1081,52 @@
       { "type": "firestarter", "moves": 100 }
     ],
     "flags": [ "LIGHT_15", "TRADER_AVOID", "ALLOWS_REMOTE_USE", "FIRESTARTER", "FIRE" ]
+  },
+  {
+    "id": "hobo_candle",
+    "type": "TOOL",
+    "name": { "str": "hobo candle" },
+    "looks_like": "jar_3l_glass_sealed",
+    "description": "An gasoline based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  You'll need a lighter or matches to light it.",
+    "weight": "965 g",
+    "volume": "3530 ml",
+    "longest_side": "21 cm",
+    "price": "20 USD",
+    "price_postapoc": "80 cent",
+    "material": [ "glass" ],
+    "symbol": "G",
+    "color": "white",
+    "ammo": [ "gasoline" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "watertight": true, "ammo_restriction": { "gasoline": 3000 } } ],
+    "use_action": {
+      "target": "hobo_candle_lit",
+      "msg": "You light the hobo candle.",
+      "active": true,
+      "need_fire": 1,
+      "need_charges": 1,
+      "menu_text": "Light",
+      "type": "transform"
+    },
+    "to_hit": { "grip": "bad", "length": "hand", "surface": "any", "balance": "clumsy" },
+    "melee_damage": { "bash": 4 }
+  },
+  {
+    "id": "hobo_candle_lit",
+    "type": "TOOL",
+    "copy-from": "hobo_candle",
+    "name": { "str": "hobo candle" },
+    "description": "An gasoline based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  This hobo candle is lit.",
```

Comment:
```suggestion
    "description": "A gasoline-based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  This hobo candle is lit.",
```